syncer/syncer.go

```diff
@@ -1641,6 +1650,7 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 		case *replication.RotateEvent:
 			err2 = s.handleRotateEvent(ev, ec)
 		case *replication.RowsEvent:
+			binlogEventRowGauge.WithLabelValues(s.cfg.WorkerName, s.cfg.Name, s.cfg.SourceID).Set(float64(ev.ColumnCount))
```

Comment:
find a correct way to get row numbers in binlog

Reply:
eyes are blurry..... my fault